### PR TITLE
CE-3335: Preserve scroll position on RWA

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
Preserve scroll position on RWA

### CODE REVIEW

File: front/main/app/routes/recent-wiki-activity.js [hunk 1]
```diff
@@ -50,6 +50,13 @@ export default Ember.Route.extend(MetaTagsMixin, {
 				label: 'recent-wiki-activity'
 			});
 			return true;
+		},
+
+		willTransition(transition) {
+			const diff = transition.params.articleDiff,
+				query = `?rc=${diff.newid}-${diff.oldid}`;
+
+			window.history.replaceState({}, null, query);
```
Comment (hakubo) [inline front/main/app/routes/recent-wiki-activity.js]: you can do so query param is bound 
and Ember can take care of that

Comment (hakubo) [inline front/main/app/routes/recent-wiki-activity.js]: https://guides.emberjs.com/v1.10.0/routing/query-params/#toc_update-url-with-code-replacestate-code-instead

File: front/main/app/templates/components/recent-wiki-activity.hbs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<header class="sub-head">
+    <div class="sub-head--title sub-head--text-only">{{i18n 'main.title' ns='recent-wiki-activity'}}</div>
+</header>
+
+<div id="recent-changes">
+	{{#each model.recentChanges as |change|}}
+		{{recent-change model=change id=change.id}}
```
Comment (hakubo) [inline front/main/app/templates/components/recent-wiki-activity.hbs]: you already pass the whole model - do we need to pass id separately?

**Comment by azarek** [comment]
LGTM :)

File: front/main/app/components/recent-wiki-activity.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+	didInsertElement() {
+		const recentChangeId = this.get('rc');
+
+		this.renderRun = true;
```
Comment (hakubo) [inline front/main/app/components/recent-wiki-activity.js]: left over?

**Comment by hakubo** [comment]
lgtm

**Pull request merged**